src/components/pin.tsx

```diff
@@ -58,7 +58,15 @@ export const Pin = (props: PropsWithChildren<PinProps>) => {
     }
 
     // Set content of Advanced Marker View to the Pin View element
-    advancedMarker.content = pinElement.element;
+    const markerContent = advancedMarker.content?.firstChild;
```

Comment:
This fixes an issue that has already been addressed in this #428 where the the Pin component would overwrite the styles on an advanced marker by setting the content rather than replace the children.

 Below is just a more verbose way of saying `...replaceChildren(pinElement.element);` since that doesn't seem to be supported by our transpile target I think.